pom.xml

```diff
@@ -260,6 +260,12 @@
         <groupId>org.apache.shiro</groupId>
         <artifactId>shiro-core</artifactId>
         <version>${shiro.version}</version>
+        <exclusions>
+          <exclusion>
```

Comment:
This version of Shiro downloads BeanUtils-1.8.3 (this is a feature that we don't use). Also, this version of BeanUtils is marked with CVE-2014-0114, hence removing this for accidental usage.